specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-08-01/frontdoor.json

```diff
@@ -0,0 +1,2484 @@
+        },
+        "frontDoorCertificateSourceParameters": {
+          "description": "Parameters required for enabling SSL with Front Door-managed certificates (if certificateSource=FrontDoor)",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/FrontDoorCertificateSourceParameters"
+        }
+      }
+    },
+    "FrontendEndpoint": {
+      "properties": {
+        "properties": {
+          "description": "Properties of the Frontend endpoint",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/FrontendEndpointProperties"
+        },
+        "name": {
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "A frontend endpoint used for routing."
+    },
+    "FrontendEndpointProperties": {
+      "description": "The JSON object that contains the properties required to create a frontend endpoint.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FrontendEndpointUpdateParameters"
+        }
+      ],
+      "properties": {
+        "resourceState": {
+          "description": "Resource status.",
+          "$ref": "#/definitions/ResourceState"
+        },
+        "customHttpsProvisioningState": {
+          "description": "Provisioning status of Custom Https of the frontendEndpoint.",
+          "readOnly": true,
+          "enum": [
+            "Enabling",
+            "Enabled",
+            "Disabling",
+            "Disabled",
+            "Failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CustomHttpsProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "customHttpsProvisioningSubstate": {
+          "description": "Provisioning substate shows the progress of custom HTTPS enabling/disabling process step by step.",
+          "readOnly": true,
+          "enum": [
+            "SubmittingDomainControlValidationRequest",
+            "PendingDomainControlValidationREquestApproval",
+            "DomainControlValidationRequestApproved",
+            "DomainControlValidationRequestRejected",
+            "DomainControlValidationRequestTimedOut",
+            "IssuingCertificate",
+            "DeployingCertificate",
+            "CertificateDeployed",
+            "DeletingCertificate",
+            "CertificateDeleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CustomHttpsProvisioningSubstate",
+            "modelAsString": true
+          }
+        },
+        "customHttpsConfiguration": {
+          "description": "The configuration specifying how to enable HTTPS",
+          "readOnly": true,
+          "$ref": "#/definitions/CustomHt
```

Comment:
not sure if these scenarios could be achieved through Azure policy. Have you checked with Policy team?